packages/contentstack-import/src/commands/cm/stacks/import.ts

```diff
@@ -109,6 +109,12 @@ export default class ImportCommand extends Command {
     'skip-audit': flags.boolean({
       description: 'Skips the audit fix.',
     }),
+    'exclude-global-modules': flags.boolean({
+      description: 'Excluded the Module that are branch independent from the import',
+    }),
+    'mapper-dir': flags.string({
+      description: 'path of backup directory for base branch import',
```

Comment:
I think we can remove the mapper-dir for now, since we dont have any module needs this